services/task-service/types/task.d.ts

```
@@ -0,0 +1,38 @@
+
+export interface CreateTask {
+    title?: string;
+    description?: string;
+}
+
+export interface TaskResponse {
+    success?: boolean;
+    task?: TaskItem;
+}
+
+export interface ListTasks {}
```

Biome

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)